java/src/jmri/jmrit/operations/locations/tools/ImportLocations.java

```diff
@@ -0,0 +1,343 @@
+ * Alternate Track, Pool name, Minimum, Track Blocking Order, Planned Pick Ups, Track Destinations, Destinations, Swap
+ * default loads and empties, Empty cars with default loads, Generate custom loads for spurs serviced by this train,
+ * Generate custom loads for any spur (multiple trains), Generate custom loads for any staging track, Block cars by pick
+ * up location, Comment, Comment when there is only pick ups, Comment when there is only set outs
+ */
+public class ImportLocations extends ImportRollingStock {
+
+    LocationManager locationManager = InstanceManager.getDefault(LocationManager.class);
+    DivisionManager divisionManager = InstanceManager.getDefault(DivisionManager.class);
+
+    private static final int FIELD_LOCATION = 0;
+    private static final int FIELD_TRACK = 1;
+    private static final int FIELD_TYPE = 2;
+    private static final int FIELD_LENGTH = 3;
+    private static final int FIELD_DIVISION = 4;
+    private static final int FIELD_SERVICED_BY = 5;
+    private static final int FIELD_ROLLING_STOCK = 6;
+    private static final int FIELD_ORDER = 7;
+    private static final int FIELD_ROAD_OPTION = 8;
+    private static final int FIELD_ROADS = 9;
+    private static final int FIELD_LOAD_OPTION = 10;
+    private static final int FIELD_LOADS = 11;
+    private static final int FIELD_SHIP_LOAD_OPTION = 12;
+    private static final int FIELD_SHIPS = 13;
+    private static final int FIELD_SET_OUT_RESTRICTIONS = 14;       // not used
+    private static final int FIELD_RESTRICTIONS_1 = 15;
+    private static final int FIELD_PICK_UP_RESTRICTIONS = 16;       // not used
+    private static final int FIELD_RESTRICTIONS_2 = 17;
+    private static final int FIELD_SCHEDULE_NAME = 18;
+    private static final int FIELD_SCHEDULE_MODE = 19;
+    private static final int FIELD_ALTERNATE_TRACK = 20;
+    private static final int FIELD_POOL_NAME = 21;
+    private static final int FIELD_IGNORE_MINIMUM = 22;
+    private static final int FIELD_TRACK_BLOCKING_ORDER = 23;
+    private static final int FIELD_PLANNED_PICK_UPS = 24;
+    private static final int FIELD_TRACK_DESTINATIONS = 25;
+    private static final int FIELD_DESTINATIONS = 26;
+    private static final int FIELD_SWAP_DEFAULT = 27;
+    private static final int FIELD_EMPTY_DEFAULT_LOADS = 28;
+    private static final int FIELD_EMPTY_CUSTOM_LOADS = 29;
+    private static final int FIELD_GENERATE_SPUR = 30;
+    private static final int FIELD_GENERATE_ANY_SPUR = 31;
+    private static final int FIELD_GENERATE_STAGING = 32;
+    private static final int FIELD_BLOCK_CARS_BY_PICKUP = 33;
+    private static final int FIELD_COMMENT = 34;
+    private static final int FIELD_COMMENT_BOTH = 35;
+    private static final int FIELD_COMMENT_PICKUPS = 36;            // not used
+    private static final int FIELD_COMMENT_SETOUTS = 37;            // not used
```

Comment:
I think it would be good if these are shared with the export tool. Otherwise the export tool might be changed while the import tool remains the same, causing the import failing to import the exported data.

@DanielBoudreau 
What's your view of this? If you think it's good as it is, I'm happy with it.